com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/flowable/FlowableFacade.java

```diff
@@ -230,6 +275,13 @@ public void deleteProcessInstance(String processInstanceId, String deleteReason)
                 // different if the process has parallel executions.
                 processEngine.getRuntimeService()
                              .setVariable(processInstanceId, Constants.PROCESS_ABORTED, Boolean.TRUE);
+                LOGGER.info("Aborting process " + processInstanceId);
+                while (true) {
```

Comment:
Extract this into method(the Thread.sleep part only). Don't we have already defined such method?